CODEOWNERS

```diff
@@ -0,0 +1,18 @@
+# Each line is a file pattern followed by one or more owners.
+
+# Order is important; the last matching pattern takes the most precedence.
+
+# When someone opens a pull request that only modifies files in directory, only
+# component owners and not the global owner(s) will be requested for a review.
+
+# Syntax: https://help.github.com/articles/about-codeowners/
+
+# Global
+* @c-parsons
+
+/skydoc @davidzchen
```

Comment:
Does david still "own" this? I don't think he has worked on this in quite some time -- we might want to consider just having me down as owner for this as well. (We can get him to chime in, certainly :))

Reply:
SGTM